.gitlab-ci.yml

```diff
@@ -153,12 +153,12 @@ spellcheck:
     stage:                           workspace
     <<:                              *docker-env
     script:
-        - cargo spellcheck check --cfg=.config/cargo_spellcheck.toml --checkers hunspell --code 1
+        - cargo spellcheck check -vvv --cfg=.config/cargo_spellcheck.toml --checkers hunspell --code 1
         - for example in examples/*/; do
-            cargo spellcheck check --cfg=.config/cargo_spellcheck.toml --checkers hunspell --code 1 ${example};
+            cargo spellcheck check -vvv --cfg=.config/cargo_spellcheck.toml --checkers hunspell --code 1 ${example};
           done
         - for contract in ${DELEGATOR_SUBCONTRACTS}; do
-            cargo spellcheck check --cfg=.config/cargo_spellcheck.toml --checkers hunspell --code 1 examples/delegator/${contract}/;
+            cargo spellcheck check -vvv --cfg=.config/cargo_spellcheck.toml --checkers hunspell --code 1 examples/delegator/${contract}/;
```

Comment:
It's much much fast if you can pass all files at once, if that does not make the stack explode.

Most time is used to load the builtin rules, if you can make that a onetime offset, it could safe you a couple of `100ms x #files`

Reply:
I took a quick look at this, and figured that adding the `--recursive` flag would do the trick, but it doesn't. Maybe an issue for another time though, haha

Reply:
I guess it traverses recursively according to the `mod` defined in all the files given, at least that's what's supposed to happen.